public/pages/workflow_detail/workspace_component/workspace_component.tsx

```diff
@@ -23,32 +21,17 @@ interface WorkspaceComponentProps {
 export function WorkspaceComponent(props: WorkspaceComponentProps) {
   const component = props.data;
 
-  const [selectedTabId, setSelectedTabId] = useState<string>('existing');
-
-  const isCreatingNew = component.allowsCreation && selectedTabId === 'new';
-  const fieldsToDisplay = isCreatingNew
-    ? component.createFields
-    : component.fields;
-
   return (
     <EuiCard title={component.label}>
       <EuiFlexGroup direction="column">
-        {/* <EuiFlexItem>
-          {component.allowsCreation ? (
-            <NewOrExistingTabs
-              setSelectedTabId={setSelectedTabId}
-              selectedTabId={selectedTabId}
-            />
-          ) : undefined}
-        </EuiFlexItem> */}
         {component.inputs?.map((input, index) => {
           return (
             <EuiFlexItem key={index}>
               <InputHandle input={input} data={component} />
             </EuiFlexItem>
           );
         })}
-        <InputFieldList inputFields={fieldsToDisplay} />
+        {/* TODO: finalize from UX what we show in the component itself. Readonly fields? Configure in the component JSON definition? */}
```

Comment:
This bring a good point, we might need to make a decision for each field for each component as some might need drop down, some might need a number and etc. might be good to add on the interface definition

Reply:
So this is actually already initially done as part of #32 where we dynamically render based on the input type. That logic is moved here into the side panel.

This comment is about what we actually show in the component itself in the reactflow workspace vs. the rendered required inputs that are in the side panel.

Reply:
We will not have inputs in the component itself per the latest UX mockups.

Reply:
I see, in the video you have above after clicking on text embedding processor, then clicking on `input field` will it open a text box or json?

Reply:
All 3 of those are text fields - see https://github.com/opensearch-project/opensearch-ai-flow-dashboards/blob/main/public/component_types/processors/text_embedding_processor.ts#L46

Maybe the confusion is that `input_field` is an input field for this example :)

Reply:
It is using https://oui.opensearch.org/1.1/#/forms/form-controls#text-field